File: Domain/Model/Comment.cs
using System;
using System.Collections.Generic;

namespace Domain;

public partial class Comment
{
    public int CommentId { get; set; }

    public int TaskId { get; set; }

    public int WrittenBy { get; set; }

    public string CommentText { get; set; }

    public DateTime WrittenAt { get; set; }

    public virtual ProjectTask Task { get; set; }

    public virtual User WrittenByNavigation { get; set; }
}

File: frontend/src/models/comment.ts
import { ProjectTask } from "./projectTask";
import { User } from "./user";

export type Comment = {
    commentId: number;
    taskId: number;
    writtenBy: number;
    commentText: string;
    writtenAt: Date;
    task: ProjectTask;
    writtenByNavigation: User;
};